pkg/backend/httpstate/client/client.go

```diff
@@ -1370,6 +1370,65 @@ func (pc *Client) SubmitAIPrompt(ctx context.Context, requestBody interface{}) (
 	return res, err
 }
 
+func getCopilotAPIPath() string {
+	return "/api/ai/chat/preview"
+}
```

Comment:
Is there a reason this is a function and not a const/var? Or pulled out at all? Looking at other functions we don't appear to do this (or at least, not consistently)

Reply:
Yeah, not consistently. But most of the other fns take args, will collapse it